registry.json

```diff
@@ -1,4 +1,16 @@
 [
+   {
+    "ticker": "CHEF",
+    "poolId": "682fd73eea4d7370b6f7a781c80b1252817b23f74688cea6257471f3",
+    "social": {
+      "website": "https://chefpool.org/",
+      "discord": "CHEF#4117",
+      "reddit": "",
+      "instagram": "",
+      "facebook": "",
+      "youtube": "https://www.youtube.com/channel/UCPPZKZi5U-KGa3RQxvQyWCw",
+      "twitter": "CardanoChefPool"
+  },
```

Comment:
You missed the section with telegram handle, operator name and github. You dont need to fill out operator name or github, but you must have the TG handle of an individual and not a group or channel